### PR TITLE
Add jenkinsfile

### PR DESCRIPTION
**Pull request by yydiamen**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+def runShellBuildStage(){
+    sh """
+        python3.11 -m venv .venv
```
Comment (thehcma) [inline Jenkinsfile:3]: I think I mentioned this before - relying on the vagaries of path is not good.

Things like the python interpreter, venv, all of these things should be defined by variables with a fixed/absolute path

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        . .venv/bin/activate
```
Comment (thehcma) [inline Jenkinsfile:4]: the same is true for the location of the virtual environment

**Review by thehcma — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (yydiamen) [reply Jenkinsfile:3]: Okay, will change, thank you.

**Comment by akrakauer** [comment]
We're deleting this branch and fixing the variables on the new-jenkins-run branch

**Pull request closed without merging**